pkg/controller/controller_utils/controller_utils_test.go

```diff
@@ -0,0 +1,35 @@
+package controller_utils
+
+import (
+	"github.com/Azure/aks-app-routing-operator/pkg/controller/testutils"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestMetricsName(t *testing.T) {
```

Comment:
We should move these tests to the same package as the ControllerName. Tests should go in the same package as the code.

Reply:
They are both in the controller_utils package.